pages/glossary/required-field.md

```diff
@@ -0,0 +1,21 @@
+---
+title: Required Field
+key: required-field
+---
+
+Any HTML element:
+
+- with one of the following [semantic roles][semantic role]: `checkbox`, `combobox`, `listbox`, `menuitemcheckbox`, `menuitemradio`, `radio`, `searchbox`, `slider`, `spinbutton`, `switch` and `textbox`;
+- that prevents the successful submission or processing of the input when empty.
+
+If the element is part of a form, having the [`required`](https://html.spec.whatwg.org/#the-required-attribute) attribute makes it a required control for form submission.
```

Comment:
The required attribute can only be set on input / select / textarea elements. Your roles allow for a lot more than that.

Reply:
reworded